docs/content/docs/test/Runner.md

```diff
@@ -17,8 +17,11 @@ It uses an internal TMS (test management system). We are working on support othe
 
 ## Getting started
 
-1. Apply the instrumentation-tests gradle plugin
-Add to your build.gradle.kts
+1. Apply the instrumentation-tests gradle plugin \
+Add to your `build.gradle.kts` or `build.gradle`
```

Comment:
I just wondering how Gradle describes it in the documentation. They include a file name in code blocks.
https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block

My proposal: move a file name to a tab description:
- Remove the line: "Add to your `build.gradle.kts` or `build.gradle`". It's obvious from samples.
- `{{< tab "build.gradle.kts" >}}`